docs/tutorial-create-pages.md

```diff
@@ -34,7 +34,7 @@ module.exports = HelloWorld;
 ```
 
 2. Go to http://localhost:3000/hello-world and you should be able to see the new page.
-1. Change the text within the `<p>...</p>` to "I can write JSX here!". The browser should refresh automatically to reflect the changes.
```

Comment:
This is intentional. Markdown will automatically increment and we don't want to manually maintain the order as it'll be hard to insert new items.